script.js

```
@@ -87,22 +365,48 @@ document.addEventListener('DOMContentLoaded', () => {
+            window.originalFetchedMenuData[category] = rawMenuData[category].map(item => ({
+                ...item,
+                originalName: item.name
+            }));
+        }
+        const preferredLanguage = localStorage.getItem('preferredLanguage') || 'es';
+        await setLanguage(preferredLanguage);
+        if (window.originalFetchedMenuData) {
+            createMenuSection(window.originalFetchedMenuData, currentLanguage);
+            initializeGalleryCarousel(window.originalFetchedMenuData, currentLanguage);
+        }
+        startAlertRotation(); // Iniciar las alertas aquí
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)